src/components/Datasets.js

```diff
@@ -188,7 +202,12 @@ class Datasets extends Component {
     }
 
     if (!datasets.size) {
-      return <Main fullscreen>{t('No datasets available')}</Main>;
+      return (
+        <Main fullscreen>
+          {t('No datasets available')}
+          {this.renderAddButton()}
+        </Main>
+      );
     }
```

Comment:
Why do we use these two conditionals (`!datasets` and `!datasets.size`)? Also, I noticed that while this component is loading, the 'No datasets available' text flashes briefly. This is not a huge deal but is not optimal. We should fix it. I can add it as a separate issue.